testapi.pm

```diff
@@ -279,9 +279,50 @@ sub upload_asset {
     return;
 }
 
+=head2 wait_screen_change
+
+  wait_screen_change { $code };
+
+wrapper around code that is supposed to change the screen. This is basically the
+opposite to wait_still_screen. Make sure to put the commands to change the screen
+within the block to avoid races between the action and the screen change
+
+  wait_screen_change {
+     send_key 'esc';
+  };
+
+=cut
+
+sub wait_screen_change(&@) {
+    my ($callback) = @_;
+
+    bmwqemu::log_call('wait_screen_change');
+
+    # get the initial screen
+    $bmwqemu::backend->set_reference_screenshot;
+    $callback->() if $callback;
+
+    my $starttime        = time;
+    my $timeout          = 10;
```

Comment:
`wait_screen_change(sub {...}, $timeout)` wouldn't work?
and
`my ($cb, $timout) = @_;` on other side?